catalogue/webapp/test/integration/stacks.test.js

```diff
@@ -0,0 +1,46 @@
+const authLoginUrl = 'https://id.wellcomecollection.org/oauth2';
+
+describe('Feature: 1. As a library member I want to know if an item is available for requesting', () => {
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    await page.setCookie({
+      name: 'toggle_stacksRequestService',
+      value: 'true',
+      path: '/',
+      domain: 'localhost',
+    });
+    await page.goto('http://localhost:3001/works/h6tcdtrt');
```

Comment:
We want to intercept this and mock the response, but there is an [unresolved issue](https://github.com/puppeteer/puppeteer/issues/5003) with puppeteer's `setRequestInterception` method, so starting with a test using the actual API.